frontend/root_words_ui/package.json

```diff
@@ -9,6 +9,10 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@emotion/styled": "^11.8.1",
```

Comment:
we forgot to add `@emotion/react` to this package.json.

Reply:
Edit: I handled this issue in [this commit](https://github.com/Quran-Journey/documents/pull/27/commits/dedec8388f166b5eada0bc8683f62a450af84f73)